README.md

```
@@ -47,28 +49,20 @@ https://docs.opencv.org/4.x/d7/d9f/tutorial_linux_install.html
-```
-dotnet new console -n ConsoleApp01
-cd ConsoleApp01
-dotnet add package OpenCvSharp4
-dotnet add package OpenCvSharp4.official.runtime.ubuntu.22.04-x64
-# -- edit Program.cs --- # 
-dotnet run
-```
-
 ### Ubuntu 24.04
 Add `OpenCvSharp4` and `OpenCvSharp4.official.runtime.ubuntu.24.04-x64` NuGet packages to your project.
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)